src/Daemon.cpp

```diff
@@ -201,7 +203,7 @@ Response dispatch_command_safe(const std::string &cmd_str, Task *task,
         Command cmd = parse_command(cmd_str);
         return dispatch_command(cmd, task, snap);
     } catch (std::runtime_error &e) {
-        std::cout << "Command failed: " << e.what() << std::endl;
+        spdlog::error("Command failed: {}", e.what());
```

Comment:
Could use some more debugging info:

```suggestion
        spdlog::error("Task {} failed: {}", task->id, e.what());
```